spec/rspec/mocks/should_syntax_spec.rb

```diff
@@ -466,36 +466,43 @@ def use_rspec_mocks
   let(:expected_arguments) {
     [
       /Using.*without explicitly enabling/,
-      {:replacement => "the new `:expect` syntax or explicitly enable `:should`"}
     ]
   }
+  let(:expected_keywords) {
+    {:replacement => "the new `:expect` syntax or explicitly enable `:should`"}
+  }
 
-  it "it warns about should once, regardless of how many times it is called" do
-    expect(RSpec).to receive(:deprecate).with(*expected_arguments)
-    o = Object.new
-    o2 = Object.new
-    o.should_receive(:bees)
-    o2.should_receive(:bees)
+  if RSpec::Support::RubyFeatures.required_kw_args_supported?
```

Comment:
Prehaps have an alternate for the older rubies? Not that fussed though as RSpec 4 won't support those.

Reply:
Okay, the old specs are restored as an alternative. https://github.com/rspec/rspec-mocks/pull/1394/commits/7c7500cb9e947a7ceb73d7cdf87924039f15031f